src/org/usfirst/frc/team2399/robot/OI.java

```diff
@@ -91,6 +93,10 @@ public static double getRightY() {
 	private static JoyClimb climbUpSpeed = new JoyClimb(RobotMap.CLIMB_UP);
 	private static JoyClimb climbDownSpeed = new JoyClimb(RobotMap.CLIMB_DOWN);
 	private static JoyClimb climbStopSpeed = new JoyClimb(RobotMap.CLIMB_STOP);
+	
+	private static AgitatorForward agitatorForwardSpeed = new AgitatorForward();
+	private static AgitatorBackward agitatorBackwardSpeed = new AgitatorBackward();
+	private static AgitatorStop agitatorStopSpeed = new AgitatorStop();
```

Comment:
Why do we need to drive the agitator in reverse or manually stop it? Shouldn't the defaultCommand handle stopping?